imxrt-hal/src/srtc.rs

```diff
@@ -0,0 +1,144 @@
+//! (Secure) Real-Time Clock
+//!
+//! Basic support only: Supports enabling and setting the clocks, but not any of their more
+//! advanced features (alarm, interrupt, power glitch detection, etc).
```

Comment:
Does the power glitch detection apply to the RTC? Or, is it one of the other SNVS functions? If it's for the SNVS, we might remove it in this RTC documentation.
```suggestion
//! advanced features (alarm, interrupt, etc).
```